{
  "url": "https://api.github.com/repos/shap/shap/issues/1187",
  "repository_url": "https://api.github.com/repos/shap/shap",
  "labels_url": "https://api.github.com/repos/shap/shap/issues/1187/labels{/name}",
  "comments_url": "https://api.github.com/repos/shap/shap/issues/1187/comments",
  "events_url": "https://api.github.com/repos/shap/shap/issues/1187/events",
  "html_url": "https://github.com/shap/shap/issues/1187",
  "id": 608432028,
  "node_id": "MDU6SXNzdWU2MDg0MzIwMjg=",
  "number": 1187,
  "title": "Possible typo in internal code",
  "user": {
    "login": "samupino",
    "id": 37086974,
    "node_id": "MDQ6VXNlcjM3MDg2OTc0",
    "avatar_url": "https://avatars.githubusercontent.com/u/37086974?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/samupino",
    "html_url": "https://github.com/samupino",
    "followers_url": "https://api.github.com/users/samupino/followers",
    "following_url": "https://api.github.com/users/samupino/following{/other_user}",
    "gists_url": "https://api.github.com/users/samupino/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/samupino/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/samupino/subscriptions",
    "organizations_url": "https://api.github.com/users/samupino/orgs",
    "repos_url": "https://api.github.com/users/samupino/repos",
    "events_url": "https://api.github.com/users/samupino/events{/privacy}",
    "received_events_url": "https://api.github.com/users/samupino/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [

  ],
  "state": "closed",
  "locked": false,
  "assignee": null,
  "assignees": [

  ],
  "milestone": null,
  "comments": 1,
  "created_at": "2020-04-28T15:59:35Z",
  "updated_at": "2023-06-04T08:35:37Z",
  "closed_at": "2023-06-04T08:35:37Z",
  "author_association": "NONE",
  "active_lock_reason": null,
  "body": "While reading the file `shap/explainers/deep/deep_tf.py` I guess I found a typo on line 117\r\n```\r\n117:   sel.fModel(cnn.inputs, cnn.get_layer(theNameYouWant).outputs)\r\n```\r\nIt is `sel.fModel` instead of `self.Model`\r\n\r\nHope it can help.",
  "closed_by": {
    "login": "thatlittleboy",
    "id": 30731072,
    "node_id": "MDQ6VXNlcjMwNzMxMDcy",
    "avatar_url": "https://avatars.githubusercontent.com/u/30731072?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/thatlittleboy",
    "html_url": "https://github.com/thatlittleboy",
    "followers_url": "https://api.github.com/users/thatlittleboy/followers",
    "following_url": "https://api.github.com/users/thatlittleboy/following{/other_user}",
    "gists_url": "https://api.github.com/users/thatlittleboy/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/thatlittleboy/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/thatlittleboy/subscriptions",
    "organizations_url": "https://api.github.com/users/thatlittleboy/orgs",
    "repos_url": "https://api.github.com/users/thatlittleboy/repos",
    "events_url": "https://api.github.com/users/thatlittleboy/events{/privacy}",
    "received_events_url": "https://api.github.com/users/thatlittleboy/received_events",
    "type": "User",
    "site_admin": false
  },
  "reactions": {
    "url": "https://api.github.com/repos/shap/shap/issues/1187/reactions",
    "total_count": 0,
    "+1": 0,
    "-1": 0,
    "laugh": 0,
    "hooray": 0,
    "confused": 0,
    "heart": 0,
    "rocket": 0,
    "eyes": 0
  },
  "timeline_url": "https://api.github.com/repos/shap/shap/issues/1187/timeline",
  "performed_via_github_app": null,
  "state_reason": "not_planned"
}
